Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -56,7 +56,9 @@ def runSelected(opt):
                      136.731, #2016B Photon data
                      140.53, #2011 HI data
                      1330, #Run2 MC Zmm
-                     135.4 #Run 2 Zee ttbar
+                     135.4, #Run 2 Zee ttbar
+                     11024.0, #2023LReco ttbar (w/ HGCal)
+                     11224.0  #2023GReco ttbar (flat tracker, Run2 calo)
```

Comment:
It's fine with me to put it only in the bot's config (I didn't realize it had a separate config for PR tests). But since we sometimes use -l limited to test PRs privately, it could be nice to have them there. I would probably prefer to add 10021.0 and 10024.0 to limited.